### ISSUE TITLE
add option to use bing AI instead of OpenAI for free use of ecoute 

### ISSUE THREAD

**Issue by mshvarts** [SevaSk/ecoute#157]
We can use ecoute for free if we could use bing AI instead of openai api key. 
Can we use this instead? i dont have a premium and it says you need to go on a waitlist to even purchase a key.
otherwise I cant use this program and I keep getting error "You exceeded your current quota, please check your plan and billing details."

https://github.com/danny-avila/LibreChat

**Comment by abhinavuppal1** [comment]
I believe one of the issues is that ecoute uses credits for getting responses from the OpenAI LLM (ChatGPT) so you end up using the free credits very quickly.

One of the forks of the repo is Transcribe - https://github.com/vivekuppal/transcribe/

It allows to turn off responses from OpenAI LLM which in turn does not require OpenAI credits, if that is what you are looking for

**Comment by j0n0w1ns** [comment]
Instead of focusing on speed, @Ashok161 you need to read. Read the conversation at least twice, before writing again

**Comment by Ashok161** [comment]
Yeah dude. You know everything. He was asking for free API key and you just pasted the link of ecoute github with the name of transcribe and didn't even answer about his doubt regarding free API key. Be it ecoute or transcribe , everything uses paid API key. If you know anything like how to use some other API key which is free other than pasting the link of existing github files , please do that. Then it makes some sense regarding what he has asked.

**Comment by mang0sw33t** [comment]
Part of the features of ecoute were available to be used for free in transcribe when this issue was filed in Nov 2023. 
And for the research of how to integrate free API's of other providers into ecoute you would have to do some research, layout what you tried and then ask specific questions.
